src/components/profile/DataClassification.vue

```diff
@@ -1,24 +1,46 @@
 <template>
-  <div :class="['data-classification', classificationClass]">
-    <span class="data-classification__label">{{ label }}</span>
-    <Tooltip
-      :buttonText="'Open ' + label + ' info'"
-      :alternateButtonText="'Close ' + label + ' info'"
-    >
-      {{ infoText }}<br />
-      {{ fluent('classification_info', 'how') }}<br />
-      <Fluent
-        id="classification_info"
-        attr="wiki"
-        :tags="{
-          wiki: {
-            tag: 'a',
-            target: '_blank',
-            href: 'https://wiki.mozilla.org/Security/Data_Classification',
-          },
-        }"
-      />
-    </Tooltip>
+  <div class="data-classification-list">
```

Comment:
I propose we refactor this into an `AccessLabel` component. Which can be configured to display data access or slack access for now. This component would then be added twice into `ViewPersonalInfo`.